panther_driver/src/panther_hardware.py

```diff
@@ -36,6 +38,11 @@ def __init__(self) -> None:
         self._setup_gpio()
         self._motor_start_sequence()
 
+        self.soft_stop_thread = threading.Thread(
+            name="soft stop proc", target=self.soft_stop
```

Comment:
`name="Soft shutdown thread"`

Reply:
:+1:

Reply:
The description should be "thread" instead of "process"

Reply:
:+1: